.gitignore

```diff
@@ -1,5 +1,7 @@
 # Binaries, binary folders
 
+binaries_*
```

Comment:
Making sure that in case someone extracts the folder in the source tree, it doesn't inadvertently gets committed.